### PR TITLE
Update configuration.md

### PR DESCRIPTION
**Pull request by suryatejaboorlu**

### CODE REVIEW

**Comment by dereknola** [comment]
You should create your config before you install or before you start the RKE2 service.

File: docs/install/configuration.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ The primary way to configure RKE2 is through its config file. Command line argum
 By default, RKE2 will launch with the values present in the YAML file located at `/etc/rancher/rke2/config.yaml`.
 
 :::note
-The RKE2 config file needs to be created manually. You can do that by running `touch /etc/rancher/rke2/config.yaml` as a privileged user.
+The RKE2 config file needs to be created manually. You can do that by running `touch /etc/rancher/rke2/config.yaml` as a privileged user. Also, RKE2 systemd service needs to be restarted after adding the config file.
```
Comment (brandond) [inline docs/install/configuration.md:12]:
```suggestion
The RKE2 config file needs to be created manually. You can do that by running `touch /etc/rancher/rke2/config.yaml` as a privileged user. If the configuration is changed after starting RKE2, the service must be restarted to apply the new configuration.
```

**Review by brandond — CHANGES_REQUESTED** [review]

**Comment by suryatejaboorlu** [comment]
Thanks @brandond. In our case we need to inject some kubectl arguments which we had to bring in in the middle of project. In this case we will definitely restart the RKE2 server for the config to get applied.

**Comment by suryatejaboorlu** [comment]
> You should create your config before you install or before you start the RKE2 service.
Thank you @dereknola. Most of the users don't know that we need to restart the rke2 service if the config file is created after sometime of deployment. In our case --pod-infra-container-image where the GC is deleting a pause image.

**Review by dereknola — APPROVED** [review]

**Review by brandond — APPROVED** [review]

**Pull request merged**